charts/spire/charts/spire-server/templates/roles.yaml

```diff
@@ -72,7 +74,6 @@ kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ include "spire-server.fullname" . }}
-  namespace: {{ .Release.Namespace }}
```

Comment:
Why are we removing this?

Reply:
Just removing namespace. its dead code. helm does that.